{{cookiecutter.project_slug}}/.travis.yml

```diff
@@ -20,11 +20,11 @@ matrix:
   fast_finish: true
 
 before_script:
-  - docker login -u={{cookiecutter.docker_username}} -p=${DOCKER_PASSWORD}
+  - docker login -u=${DOCKER_USERNAME} -p=${DOCKER_PASSWORD}
```

Comment:
We could also hard code this to `decaftravis` in order to create less incentives to put in personal credentials?